src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -147,9 +147,12 @@ function WorkspaceMembersPage({
     }, [selectedEmployees, policy?.owner, session?.accountID]);
 
     useEffect(() => {
+        if (!isFocused) {
+            return;
+        }
         getWorkspaceMembers();
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
+    }, [isFocused]);
```

Comment:
Why is this the only page where we do not use the `useFocusEffect`? Can we use it so its consistent?

Reply:
@tienifr thanks, can you leave a comment about that in the code?